Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpUnitTestsRunnerExecutionStrategy.cs

```diff
@@ -0,0 +1,203 @@
+﻿namespace OJS.Workers.ExecutionStrategies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Text.RegularExpressions; 
+      
+    using Microsoft.Build.Evaluation;
+
+    using OJS.Common.Extensions;
+    using OJS.Common.Models;
+    using OJS.Workers.Checkers;
+    using OJS.Workers.Common;
+    using OJS.Workers.Executors;
+   
+    public class CSharpUnitTestsRunnerExecutionStrategy : ExecutionStrategy
+    {
+        private const string ZippedSubmissionName = "Submission.zip";
+        private const string TestedCode = "TestedCode.cs";
+        private const string CsProjFileSearchPattern = "*.csproj";
+        private const string ProjFileExtenstion = ".csproj";
```

Comment:
IMO rename to `ProjectFileExtenstion` -> you do not know that it is a `proj `file here, before defining this variable, so... Please rename in all similar places @KristianMariyanov are u o.k. with this?